plugins/inputs/ipmi_sensor/ipmi.go

```diff
@@ -93,23 +102,32 @@ func (m *Ipmi) parse(acc telegraf.Accumulator, server string) error {
 		opts = conn.options()
 	}
 	opts = append(opts, "sdr")
+	if m.SchemaVersion == 2 {
+		opts = append(opts, "elist")
+	}
 	cmd := execCommand(m.Path, opts...)
 	out, err := internal.CombinedOutputTimeout(cmd, m.Timeout.Duration)
 	if err != nil {
 		return fmt.Errorf("failed to run command %s: %s - %s", strings.Join(cmd.Args, " "), err, string(out))
 	}
+	if m.SchemaVersion == 2 {
+		return parseV2(acc, hostname, string(out))
+	}
+	return parseV1(acc, hostname, string(out))
+}
 
+func parseV1(acc telegraf.Accumulator, hostname string, cmdOut string) error {
 	// each line will look something like
 	// Planar VBAT      | 3.05 Volts        | ok
-	lines := strings.Split(string(out), "\n")
+	lines := strings.Split(cmdOut, "\n")
```

Comment:
I'll bet this is was an existing bug in Windows due to only allowing CR line endings.  What would be ideal is if we switched to bufio.Scanner here.